orb/core/connection_types/sql/postgres/statements/select.py

```diff
@@ -143,63 +143,70 @@ def __call__(self, model, context, fields=None):
         # generate sql statements
         try:
             sql_where, sql_where_data = WHERE(model, where, fields=fields)
+
+        # null queries should not result in any results
+        # examples of this scenario would be query like
+        #    orb.Query('id').in_([])
+        # this query would obviously not ever return results, so no need
+        # to ever query it
         except orb.errors.QueryIsNull:
-            sql_where, sql_where_data = '', {}
```

Comment:
this used to return this as the results...now making sure it does - a NULL query should just cancel all other operations because some condition will mean it will never find any results